envi/__init__.py

```diff
@@ -156,6 +156,26 @@ def archGetRegisterGroups(self):
         allr = [rname for rname in regctx.getRegisterNames()]
         return [ ('all', allr), ]
 
+    def archModifyFuncAddr(self, va, info):
+        '''
+        Can modify the VA and context based on architecture-specific info.
+        Default: return the same va, info
+
+        This hook allows an architecture to correct VA and Architecture, such 
+        as is necessary for ARM/Thumb.
+        '''
+        return va, info
```

Comment:
so what needs to be changed here?  if we're passing it straight through, is it ok to not create a copy?
we're already returning the dict.  arm currently updates the same dict but returns it.  i have a delta ready to make that create a copy if something needs to change.  what do you think?